Makefile

```diff
@@ -601,9 +606,10 @@ update-dependencies:
 	@go get -u -t $(PINNED_DEPENDENCIES) ./...
 	@go mod tidy
 
-go-generate: $(MOCKGEN) $(GOIMPORTS) $(STRINGER)
+go-generate: $(GOWRAP) $(MOCKGEN) $(GOIMPORTS) $(STRINGER)
 	@printf $(COLOR) "Process go:generate directives..."
 	@go generate ./...
+	$(MAKE) copyright
```

Comment:
Without this addition, the generated code has no copyright header. Fortunately, this step is idempotent (and fast).

... actually, I think none of the code generators should bother with the copyright header themselves and just rely on this being invoked tbh.